python/README.md

```diff
@@ -0,0 +1,62 @@
+# python
+
+A Wing library for integrating with Python code.
+
+## Prerequisites
+
+* [winglang](https://winglang.io).
+
+## Installation
+
+```sh
+npm i @winglibs/python
+```
+
+## Usage
+
+```js
+bring python;
+
+let func = new python.Function(
+  path: "./test-assets",
+  handler: "main.handler",
+);
+
+test "invokes the function" {
+  let res = func.invoke();
+}
+```
+
+It is also possible to interact with Wing resources through the python code
+
+```js
+// main.w
+let bucket = new cloud.Bucket();
+let func = new python.Function(
+  path: "./test-assets",
+  handler: "main.handler",
+);
+
+func.liftClient("bucket", bucket, ["get", "put"]);
```

Comment:
I think `lift()` should be sufficient:

```suggestion
func.lift("bucket", bucket, ["get", "put"]);
```

I think we can also use keywords to make this nicer:

```js
func.lift(bucket, as: "my_bucket", allow: ["get", "put"]);
```

What do you think?

Reply:
I like this API suggestion. 
@eladcon, once you move forward lets also make these API  changes to tsoa